src/uct/api/uct_def.h

```diff
@@ -210,6 +210,176 @@ typedef struct uct_cm_remote_data {
 } uct_cm_remote_data_t;
 
 
+/**
+ * @ingroup UCT_CLIENT_SERVER
+ * @brief Server endpoint's connect callback arguments field mask.
+ *
+ * The enumeration allows specifying which fields in
+ * @ref uct_cm_ep_server_connect_cb_handle are present, for backward compatibility
+ * support.
+ */
+enum uct_cm_ep_server_connect_cb_handle_field {
```

Comment:
enum uct_cm_ep_server_connected_args_field {
}

Reply:
why "connected"? we call this callback with error status as well...